pkg/types/types.go

```diff
@@ -69,6 +70,14 @@ type RuntimeConfig struct {
 	CNIDeviceInfoFile string          `json:"CNIDeviceInfoFile,omitempty"`
 }
 
+// LogOptions specifies the configuration of the log
+type LogOptions struct {
+	MaxAge     int  `json:"maxAge,omitempty"`
+	MaxSize    int  `json:"maxSize,omitempty"`
+	MaxBackups int  `json:"maxBackups,omitempty"`
+	Compress   bool `json:"compress,omitempty"`
+}
```

Comment:
Could you please provide document for new option, in Multus-cni Configuration Reference
, https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/configuration.md?

Reply:
Sure.